src/CreateReadout.hpp

```diff
@@ -20,6 +20,12 @@
 #include "ReadoutConcept.hpp"
 #include "ReadoutModel.hpp"
 
+#include "WIBRequestHandler.hpp"
```

Comment:
I'm wondering if we could a preprocessor trick to collect specializations from an include location....